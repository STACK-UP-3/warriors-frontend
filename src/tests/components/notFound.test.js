import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/noFound';

describe('*************** Testing the Not Found  component ***************',()=>{
    it('Should render the reset password page correctly', ()=>{
        const wrapper = shallow( <NotFound  title={'item'} /> );
        expect(wrapper).toMatchSnapshot();
    });
});