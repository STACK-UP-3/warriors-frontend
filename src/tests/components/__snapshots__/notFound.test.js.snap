// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`*************** Testing the Not Found  component *************** Should render the reset password page correctly 1`] = `
<div>
  <h1>
    404: Can not Find The item 
    item
  </h1>
  <p>
    The requested Item can not be found in our database.
  </p>
</div>
`;
